x-pack/plugins/xpack_main/server/lib/feature_registry/feature_registry.ts

```diff
@@ -55,13 +55,19 @@ const prohibitedFeatureIds: Array<keyof UICapabilities> = ['catalogue', 'managem
 
 const featurePrivilegePartRegex = /^[a-zA-Z0-9_-]+$/;
 const managementSectionIdRegex = /^[a-zA-Z0-9_-]+$/;
+
+// These expressions are intentionally identical because they all eventually transform in to UI Capabilities downstream, so must at least confirm to that.
```

Comment:
We were initially using different variables for these because they could potentially have "different reasons to change". In this situation, it feels like we should probably be reusing the `uiCapabilitiesRegex`, since we know that these will turn into ui capabilities, and therefore have the "same reason to change".

Reply:
works for me!